### PR TITLE
Add Vidar as new Jupyter Server SSC Representative

### PR DESCRIPTION
**Pull request by Zsailer**
cc @vidartf

### CODE REVIEW

**Review by JohanMabille — APPROVED** [review]

**Comment by JohanMabille** [comment]
Thanks!

File: about.html [hunk 1]
```diff
@@ -371,6 +371,12 @@
     gh_handle: marthacryan
     linkedin: https://www.linkedin.com/in/marthacryan/
     subproject: DEI Working Group  
+  - name: Vidar Fauske
```
Comment (vidartf) [inline about.html:374]:
```suggestion
  - name: Vidar Fauske
```
```suggestion
  - name: Vidar T. Fauske
```

**Review by vidartf — APPROVED** [review]
Approved with minor naming tweak :+1:

**Review by jasongrout — APPROVED** [review]
Awesome, thanks!

**Comment by jasongrout** [comment]
Looks like the check links test is failing for unrelated reasons.

**Pull request merged**